### ISSUE TITLE
[Icons] icons are not imported anymore with AllIcons.

### ISSUE THREAD

**Issue by amaillard-sap** [SAP/ui5-webcomponents#8644]
### Bug Description

As of 1.23.2 icons are not loaded properly when using generic import 

```
import '@ui5/webcomponents-icons/dist/AllIcons.js';
```

### Affected Component

icon

### Expected Behaviour

All icons should be imported as before (was fine on 1.23.1)

### Isolated Example

https://stackblitz.com/edit/js-mgpyrt?file=index.js

### Steps to Reproduce

1. Same code as before, using AllIcons import
2. default icons are not imported

icons imported explicitly are still working

### Priority

Very High

### UI5 Web Components Version

1.23.2

### Browser

Chrome

### Additional Context

We will not be able to use the 1.23.2 release for SBPA since it break the application visual.

### Organization

SPBA

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ilhan007** [comment]
Hi @amaillard-sap the 1.23.2 has not been properly built, we are working on new patch, we will update you shortly

**Comment by ilhan007** [comment]
Hi @amaillard-sap can you try 1.23.3

**Comment by amaillard-sap** [comment]
I updated the stack blitz and the error is still there, now with a log message:

```
Module ../generated/assets/v5/SAP-icons.json not declared as a System.registerDynamic dependency of https://js-mgpyrt.stackblitz.io/turbo_modules/@ui5/webcomponents-icons@1.23.3/dist/json-imports/Icons.js
```

**Comment by amaillard-sap** [comment]
Fixed with 1.24.0, yu can close this issue (and maybe deprecated 1.23.2 & 1.23.3)
